### PR TITLE
Add Spam Detection to docs.mdx

### PR DESCRIPTION
**Pull request by jas-kas**
Includes:
- Manual marking as spam
- Auto-spam detection

### CODE REVIEW

File: docs/product/user-feedback/index.mdx [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can either manually resolve user feedback submissions on the **User Feedback
 
 ![User Feedback List Triaging.](./img/user-feedback-triaging.png)
 
-You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view.
+You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. For user feedback submissions you consider spam, you can mark them as such selecting the feedback(s) and clicking "Mark as Spam". These user feedback submissions will then appear under the Spam heading.
```
Comment (michellewzhang) [inline docs/product/user-feedback/index.mdx:55]:
```suggestion
You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. For user feedback submissions you consider spam, you can mark them as such by selecting the feedback(s) and clicking "Mark as Spam". These user feedback submissions will then appear under the Spam heading. 
```

File: docs/product/user-feedback/index.mdx [hunk 2]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely spam and places them inside the “Spam” folder on the **User Feedback** page to improve the quality and actionability of feedbacks in the main list. Feedbacks detected as spam will also bypass any Alert Rules you have configured for User Feedback so you only get alerted when it matters.
+
+In order to do auto-spam detection, we use a Large Language Model to classify messages. We are using a language model internal to Google Cloud Platform where no data is stored or persisted. This is part of our existing [[sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors)](https://sentry.io/legal/dpa/#subprocessors).
```
Comment (michellewzhang) [inline docs/product/user-feedback/index.mdx:68]: is the two links here intentional?

File: docs/product/user-feedback/index.mdx [hunk 3]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
...
+
+If you’d like to opt out of Spam Detection, turn off "Enable Spam Detection" in Settings > Projects > [Project Name] > User Feedback (Under “Processing”).
```
Comment (michellewzhang) [inline docs/product/user-feedback/index.mdx:70]:
```suggestion
If you’d like to opt out of spam Ddtection, turn off "Enable Spam Detection" in Settings > Projects > [Project Name] > User Feedback (under “Processing”).
```

**Review by michellewzhang — APPROVED** [review]

File: docs/product/user-feedback/index.mdx (hunk 2 above)
Comment (jas-kas) [reply docs/product/user-feedback/index.mdx:68]: No, copying from Notion did my dirty

Comment (jas-kas) [inline docs/product/user-feedback/index.mdx:68]:
```suggestion
In order to do auto-spam detection, we use a Large Language Model to classify messages. We are using a language model internal to Google Cloud Platform where no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
```

File: docs/product/user-feedback/index.mdx [hunk 4]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely spam and places them inside the “Spam” folder on the **User Feedback** page to improve the quality and actionability of feedbacks in the main list. Feedbacks detected as spam will also bypass any Alert Rules you have configured for User Feedback so you only get alerted when it matters.
+
+In order to do auto-spam detection, we use a Large Language Model to classify messages. We are using a language model internal to Google Cloud Platform where no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
```
Comment (jas-kas) [inline docs/product/user-feedback/index.mdx:68]: @michellewzhang fixed!

Comment (michellewzhang) [reply docs/product/user-feedback/index.mdx:68]: looks good! thanks

File: docs/product/user-feedback/index.mdx [hunk 5]
```diff
@@ -52,7 +52,7 @@ You can either manually resolve user feedback submissions on the **User Feedback
 
 ![User Feedback List Triaging.](./img/user-feedback-triaging.png)
 
-You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view.
+You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. For user feedback submissions you consider spam, you can mark them as such by selecting the feedback(s) and clicking "Mark as Spam". These user feedback submissions will then appear under the Spam heading.
```
Comment (lizokm) [inline docs/product/user-feedback/index.mdx:55]:
```suggestion
You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. If you come across user feedback that you consider spam, select the feedback(s) and click "Mark as Spam". This feedback will then be filtered out and appear under the Spam heading. 
```

File: docs/product/user-feedback/index.mdx [hunk 6]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely spam and places them inside the “Spam” folder on the **User Feedback** page to improve the quality and actionability of feedbacks in the main list. Feedbacks detected as spam will also bypass any Alert Rules you have configured for User Feedback so you only get alerted when it matters.
+
```
Comment (lizokm) [inline docs/product/user-feedback/index.mdx:67]:
```suggestion
By default, Sentry auto-detects user feedback submissions that are likely to be spam and puts them in the “Spam” folder on the **User Feedback** page. This improves the quality and actionability of the feedback that's in the main list. To improve the quality of your alerts, feedback that's labeled as spam will bypass any User Feedback Alert Rules you may have configured.

```

File: docs/product/user-feedback/index.mdx (hunk 4 above)
Comment (lizokm) [inline docs/product/user-feedback/index.mdx:68]:
```suggestion
To make auto-spam detection possible, we use a Large Language Model that classifies messages. This language model is internal to Google Cloud Platform and no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
```

File: docs/product/user-feedback/index.mdx [hunk 7]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
...
+In order to do auto-spam detection, we use a Large Language Model to classify messages. We are using a language model internal to Google Cloud Platform where no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
+
+If you’d like to opt out of spam detection, turn off "Enable Spam Detection" in Settings > Projects > [Project Name] > User Feedback (under “Processing”).
```
Comment (lizokm) [inline docs/product/user-feedback/index.mdx:70]:
```suggestion
If you’d like to opt out of spam detection, turn off "Enable Spam Detection" in [Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/) > Projects > [Project Name] > User Feedback (under “Processing”).
```

**Review by lizokm — APPROVED** [review]
Just a few small tweaks :)

**Comment by jas-kas** [comment]
thank you!! @lizokm 🙏 

https://openseauserdata.com/files/7d5d665f92c09858439458678bc879a4.gif

File: docs/product/user-feedback/index.mdx [hunk 8]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely to be spam and puts them in the “Spam” folder on the **User Feedback** page. This improves the quality and actionability of the feedback that's in the main list. To improve the quality of your alerts, feedback that's labeled as spam will bypass any User Feedback Alert Rules you may have configured.
```
Comment (jas-kas) [inline docs/product/user-feedback/index.mdx:66]:
```suggestion
By default, Sentry auto-detects user feedback submissions that are likely to be spam and puts them in the “Spam” folder on the **User Feedback** page. This improves the quality and actionability of the feedback that are in the main list. To improve the quality of your alerts, feedback that's labeled as spam will bypass any User Feedback Alert Rules you may have configured.
```

File: docs/product/user-feedback/index.mdx [hunk 9]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely to be spam and puts them in the “Spam” folder on the **User Feedback** page. This improves the quality and actionability of the feedback that are in the main list. To improve the quality of your alerts, feedback that's labeled as spam will bypass any User Feedback Alert Rules you may have configured.
+
+To make auto-spam detection possible, we use a Large Language Model that classifies messages. This language model is internal to Google Cloud Platform and no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
+
+If you’d like to opt out of spam detection, turn off "Enable Spam Detection" in [Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/) > Projects > [Project Name] > User Feedback (under “Processing”).
```
Comment (jas-kas) [inline docs/product/user-feedback/index.mdx:70]:
```suggestion
If you’d like to opt out of spam detection, turn off "Enable Spam Detection" in [Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/) > Projects > [Project Name] > User Feedback (under “Project”).
```

File: docs/product/user-feedback/index.mdx [hunk 10]
```diff
@@ -52,7 +52,7 @@ You can either manually resolve user feedback submissions on the **User Feedback
 
 ![User Feedback List Triaging.](./img/user-feedback-triaging.png)
 
-You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view.
+You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. If you come across user feedback that you consider spam, select the feedback(s) and click "Mark as Spam". This feedback will then be filtered out and appear under the Spam heading.
```
Comment (vivianyentran) [inline docs/product/user-feedback/index.mdx:55]:
```suggestion
You can also assign a team member to a specific user feedback submission on the **User Feedback Details** view. If you come across user feedback that you consider spam, select the feedback(s) and click "Mark as Spam". This feedback will then be filtered out and moved to the "Spam" heading. 
```

File: docs/product/user-feedback/index.mdx [hunk 11]
```diff
@@ -62,6 +62,15 @@ Example search: `url:*/payments/*`
 
 For a list of search fields, refer to the [search documentation](https://docs.sentry.io/product/reference/search/searchable-properties/user-feedback/).
 
+## Spam Detection for User Feedback
+By default, Sentry auto-detects user feedback submissions that are likely to be spam and puts them in the “Spam” folder on the **User Feedback** page. This improves the quality and actionability of the feedback that are in the main list. To improve the quality of your alerts, feedback that's labeled as spam will bypass any User Feedback Alert Rules you may have configured.
+
+To make auto-spam detection possible, we use a Large Language Model that classifies messages. This language model is internal to Google Cloud Platform and no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
```
Comment (vivianyentran) [inline docs/product/user-feedback/index.mdx:68]:
```suggestion
To make automatic spam detection possible, we use a Large Language Model that classifies messages. This language model is internal to Google Cloud Platform and no data is stored or persisted. This is part of our existing [sub-processor agreement](https://sentry.io/legal/dpa/#subprocessors).
```

**Review by vivianyentran — APPROVED** [review]
looks good!

**Pull request merged**